src/main/java/cubicchunks/world/cube/Cube.java

```diff
@@ -67,18 +71,22 @@
 
 	//used to track if the cube should be unloaded or not, done instead of removing cube from
 	//unloadQueue each time something loads it
-	public boolean unloaded;
+	public  boolean unloaded;
+
+	private boolean isModified            = false;
+	private boolean isPopulated           = false;
+	private boolean isFullyPopulated      = false;
+	private boolean isInitialLightingDone = false;
```

Comment:
What is the benefit of this? All of them are initialized to false. You can clearly see that even if they are not lines up.